lib/Routing/Staticroute/Router.php

```diff
@@ -124,7 +124,7 @@ public function getRouteDebugMessage($name, array $parameters = [])
     /**
      * @inheritDoc
      */
-    public function generate($name, $parameters = [], $referenceType = self::ABSOLUTE_PATH)
+    public function generate($name, $parameters = [], $referenceType = self::ABSOLUTE_PATH, $encode = true)
```

Comment:
Unfortunately this isn't possible because of `\Symfony\Component\Routing\Generator\UrlGeneratorInterface::generate($name, $parameters = [], $referenceType = self::ABSOLUTE_PATH)` 


So we'd have to stick to the `$parameters` approach, and fix it this way. 

Can you please update your PR? 
Thanks!

Reply:
I changed the "encode" to $parameters.